packages/api/db/migration/20250520185926_create_soil_sample_task_table.js

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright 2025 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export const up = async function (knex) {
+  await knex.schema.createTable('soil_sample_task', (table) => {
+    table.integer('task_id').references('task_id').inTable('task').primary();
+    table
+      .uuid('document_id')
```

Comment:
I added a comment in the Jira doc -- I think I heard it should be possible to upload multiple docs to the same task, in which case we'll need a separate table instead of a column (still pending confirmation)